demisto_sdk/commands/common/handlers/json/ujson_handler.py

```diff
@@ -31,25 +31,29 @@ def load(self, fp: IO[str]):
         except ValueError as e:
             raise JSONDecodeError(e)
 
-    def dump(self, obj: Any, fp: IO[str], indent=0, sort_keys=False):
+    def dump(self, obj: Any, fp: IO[str], indent=0, sort_keys=False, **kwargs):
+        escape_forward_slashes = kwargs.pop('escape_forward_slashes', False)
         try:
             ujson.dump(
                 obj,
                 fp,
                 indent=indent,
                 sort_keys=sort_keys,
-                escape_forward_slashes=False
+                escape_forward_slashes=escape_forward_slashes,
+                **kwargs
```

Comment:
We don't want to allow `kwargs`. We want to fully control the allowed arguments for the `dump` or `load` methods.
Just add the argument `escape_forward_slashes` which defaults to `None`.

Reply:
@ilaner ?

Reply:
Decided along with @ilaner to use kwargs on abstract methods, but on the implementing methods use the actual variables we want to support